### PR TITLE
38 ng style login forms

### LINKED ISSUE #38: Style login/create account forms

### PR DESCRIPTION
**Pull request by njgheorghita**
Completed styling for create an account form and login form.
Maintained similar style from the home page form. 

closes #38

### CODE REVIEW

File: app/views/sessions/new.html.haml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-  %br
-  = f.text_field :email
-  %br
-  = f.label :password
-  %br
-  = f.password_field :password
-  %br
-  = f.submit "Login"
\ No newline at end of file
+%section.account_form
```
Comment (tmikeschu) [inline app/views/sessions/new.html.haml:1]: Solid HAML setup 👌

**Comment by ski-climb** [comment]
The form looks great!  Thanks @njgheorghita :shipit:

File: app/views/users/new.html.haml [hunk 2]
```diff
@@ -1,28 +1,18 @@
...
-  = f.password_field :password
-  %br
-  = f.label :password_confirmation
-  %br
-  = f.password_field :password_confirmation
-  %br
-  = f.submit 'Sign Up'
+%section.account_form  
+  %h1 Create your Pad Crash Account
+  %br
```
Comment (tmikeschu) [inline app/views/users/new.html.haml:3]: In the future, we can probably achieve this spacing using the flex box properties.

**Review by tmikeschu — COMMENTED** [review]
Nice work, thanks @njgheorghita

**Pull request merged**